recipes/swapfile.rb

```diff
@@ -1,13 +1,27 @@
 #
-# Author:: Carl Antuar (<carl.antuar@smartservice.qld.gov.au>)
+# Author:: Carl Antuar (<carl.antuar@smartservice.qld.gov.au>), William Dutton (<will.dutt@gmail.com>)
 # Cookbook Name:: datashades
 # Recipe:: swapfile
 #
-# Sets up a swapfile on the extra EBS, if available.
-# Since this isn't on the root filesystem and may not be instantly
-# available on startup, it's not added to /etc/fstab.
+# Sets up a swapfile on the root disk
+# Setup another swap on external disk for more speed if it is used during normal operations
```

Comment:
It's not an extra disk anymore, it's the root.

Reply:
I'm still adding the extra swap for extra performance

https://wiki.archlinux.org/title/swap

If two or more areas have the same priority, and it is the highest priority available, pages are allocated on a round-robin basis between them.

Reply:
I don't see how that will improve performance? A 4GB swapfile is already overkill for every use case we've encountered. Adding even more than that seems pointless.